include/gatekeeper_net.h

```diff
@@ -497,6 +500,22 @@ hw_filter_eth_available(const struct gatekeeper_if *iface)
 	return iface->hw_filter_eth && iface->rss;
 }
 
+static inline void
+set_ipv4_checksum(struct gatekeeper_if *iface, struct rte_mbuf *pkt,
```

Comment:
Are there requirements before calling this function? For example, having fields `l3_len` and `l2_len` of  `pkt` been filled in? Identifying these requirements will be helpful for future uses of `set_ipv4_checksum()`.